x-pack/plugins/security_solution/public/resolver/store/data/selectors.test.ts

```diff
@@ -220,14 +220,17 @@ describe('data state', () => {
     it('should be loading', () => {
       expect(selectors.isTreeLoading(state())).toBe(true);
     });
+    it('should not have a request to abort', () => {
+      expect(selectors.treeRequestParametersToAbort(state())).toBe(null);
+    });
     it('should not have an error, more children, more ancestors, a request to make, or a pending request that should be aborted.', () => {
       expect(viewAsAString(state())).toMatchInlineSnapshot(`
         "is loading: true
         has an error: false
         has more children: false
         has more ancestors: false
-        parameters to fetch: {\\"databaseDocumentID\\":\\"databaseDocumentID\\",\\"indices\\":[],\\"filters\\":{},\\"dataRequestID\\":0}
-        requires a pending request to be aborted: {\\"databaseDocumentID\\":\\"databaseDocumentID\\",\\"indices\\":[],\\"filters\\":{}}"
+        parameters to fetch: null
```

Comment:
Are these `null`s correct? I need to go back and look at how they were before the original PR but would this still have the `filters` and `databaseDocumentID` etc?